USAGE.md

```diff
@@ -344,3 +346,15 @@ Users leveraging this feature should be comfortable with the internals of the Ap
 familiarity with the
 [`MantaConnectionFactory`](/java-manta-client-unshaded/src/main/java/com/joyent/manta/http/MantaConnectionFactory.java)
 class is recommended.
+
+### Modifying configuration after client instantiation
+
+Some configuration parameters can be updated in the `ConfigContext` after the client has already been constructed.
+Instead of directly providing a `ConfigContext` users can instead supply a `AuthAwareConfigContext` to the `MantaClient`
+constructor which can be programmatically updated using its `reload()` method. Because the `MantaClient` can be used in a
+multi-threaded fashion, users should be careful to await termination of in-flight requests in before triggering authentication reload.
```

Comment:
If users need to stop all requests, call `reload()`, and then start up again, when would I choose this over just creating a new `MantaClient`?

Reply:
Attempting to provide flexibility for interactive applications. In the case of Cyberduck the client objects are created before all the authentication parameters are available.